deploy/docker/clickhouse-setup/otel-collector-config.yaml

```
@@ -70,6 +70,40 @@ receivers:
 
 
 processors:
+  logstransform/default:
```

```suggestion
  logstransform/internal:
```

same suggestion as the charts PR.

---

Done